server/prometheus.js

```diff
@@ -37,25 +38,58 @@ class Prometheus {
 
     /**
      * @param {object} monitor Monitor object to monitor
+     * @param {Array<object>} tags Tags to add to the monitor
```

Comment:
I think this is the proper way to type this, or?

```suggestion
     * @param {Array<{name:string,value:string}>} tags Tags to add to the monitor
```